src/pages/components/dropdown/style.mdx

```diff
@@ -244,13 +251,14 @@ be the same width and appropriately fit the design, layout, and content.
 
 ### Combo box
 
-| Element      | Property      | px / rem | Spacing token |
-| ------------ | ------------- | -------- | ------------- |
-| Label        | margin-bottom | 8 / 0.5  | `$spacing-03` |
-| Field        | padding-left  | 16 / 1   | `$spacing-05` |
-|              | padding-right | 72 / 4.5 | –             |
-| Chevron icon | padding-right | 16 / 1   | `$spacing-05` |
-| Clear icon   | padding-right | 8 / 0.5  | `$spacing-03` |
+| Element          | Property                    | px / rem | Spacing token |
+| ---------------- | --------------------------- | -------- | ------------- |
+| Label            | margin-bottom               | 8 / 0.5  | `$spacing-03` |
+| Field            | padding-left                | 16 / 1   | `$spacing-05` |
+|                  | padding-right               | 72 / 4.5 | –             |
+| Chevron icon     | padding-right               | 16 / 1   | `$spacing-05` |
+| Vertical divider | padding-right, padding-left | 16 / 1   | `$spacing-05` |
```

Comment:
I think the Vertical divider padding-right, padding-left is actually 8/.5 $spacing-03, according to the image? There is 8px spacing to the left and to the right of the divider.

Reply:
Yes you're right
I'll correct it, thanks Lauren!